node_modules/oae-core/filepreview/filepreview.html

```diff
@@ -54,6 +54,32 @@
     </div>
 --></div>
 
+<!-- VIDEO PREVIEW -->
+<div id="filepreview-video-template"><!--
+    <div id="filepreview-video-container" class="text-center">
+        {if content.previews && content.previews.video}
+            {var url = content.previews.video.replace('remote:', '')}
+            {var mime = 'video/mp4'}
+        {else}
+            {var url = content.downloadPath}
```

Comment:
This is something we tried in the past, and the results were very inconsistent across browsers/user machines so it might be safest to not show the preview unless we were able to generate one through ZenCoder.